tests/test_core.py

```diff
@@ -7159,30 +7159,28 @@ def encode_utf8(data):
     build_and_check()
 
   def test_modularize_closure_pre(self):
-    emcc_args = self.emcc_args[:]
-    for instance in [0, 1]:
-      print(instance)
-      # test that the combination of modularize + closure + pre-js works. in that mode,
-      # closure should not minify the Module object in a way that the pre-js cannot use it.
-      self.emcc_args = emcc_args + [
-        '--pre-js', path_from_root('tests', 'core', 'modularize_closure_pre.js'),
-        '--closure', '1',
-        '-g1'
-      ]
-      if not instance:
-        self.emcc_args += ['-s', 'MODULARIZE=1']
+    # test that the combination of modularize + closure + pre-js works. in that mode,
+    # closure should not minify the Module object in a way that the pre-js cannot use it.
+    base_args = self.emcc_args[:] + [
```

Comment:
I believe the `[:]` is not necessary here, as Python creates a new object due to the `+` anyhow.

Reply:
Nice catch!